src/visualizations/config/generators/highcharts/index.js

```diff
@@ -3,16 +3,24 @@ import HM from 'highcharts/highcharts-more'
 import HB from 'highcharts/modules/boost'
 import HE from 'highcharts/modules/exporting'
 import HNDTD from 'highcharts/modules/no-data-to-display'
+import HOE from 'highcharts/modules/offline-exporting'
 import HPF from 'highcharts/modules/pattern-fill'
 import HSG from 'highcharts/modules/solid-gauge'
+import PEBFP from './pdfExportBugFixPlugin/index.js'
 
 // apply
 HM(H)
 HSG(H)
 HNDTD(H)
 HE(H)
+HOE(H)
 HPF(H)
 HB(H)
+PEBFP(H)
+
+/* Whitelist some additional SVG attributes here. Without this,
+ * the PDF export for the SingleValue visualization breaks. */
+H.AST.allowedAttributes.push('fill-rule', 'clip-rule')
```

Comment:
This is another potential breaking point as we play with Highcharts internals.
It happened before that after updating Highcharts stuff broke because they changed something internally.
Ideally we shouldn't have code like this, it's hard to predict a problem even if we read carefully the changelog where changes to internal stuff is not necessarily visible there.

In some cases we don't have an alternative solution tho.
We should just keep in mind these parts whenever we update Highcharts, and test.